jib-gradle-plugin/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Now disabling color in `plain` console output. ([#3087](https://github.com/GoogleContainerTools/jib/pull/3087))
```

Comment:
We usually choose a link that has better description and more detailed information.

And I think this can be confusing, since we already disables color. How about this?

```suggestion
- Fixed an issue that some log used color in "plain" console output. ([#2764](https://github.com/GoogleContainerTools/jib/pull/2764))
```